dotnet/src/Functions/Functions.OpenApi/RestApiOperationRunner.cs

```diff
@@ -214,7 +214,7 @@ private async Task<RestApiOperationResponse> SendAsync(
 
         try
         {
-            responseMessage = await this._httpClient.SendWithSuccessCheckAsync(requestMessage, cancellationToken).ConfigureAwait(false);
+            responseMessage = await this._httpClient.SendWithSuccessCheckAsync(requestMessage, HttpCompletionOption.ResponseHeadersRead, cancellationToken).ConfigureAwait(false);
```

Comment:
@stephentoub do you see any issues with applying the options to all HTTP requests?

Reply:
With applying ResponseHeadersRead? No, that's always fine. Under the covers at the HttpMessageHandler layer there's zero difference, everything is the equivalent of ResponseHeadersRead. ResponseContentRead is just a nicety provided at the HttpClient layer that buffers the response stream from the handler.